requirements/deploy.txt

```diff
@@ -234,7 +234,7 @@ markdown==3.4.4
     # via -r requirements/pip.txt
 matplotlib-inline==0.1.6
     # via ipython
-newrelic==8.10.1
+newrelic==8.11.0
```

Comment:
It seems there is a first party support for structlog now: https://docs.newrelic.com/docs/release-notes/agent-release-notes/python-release-notes/python-agent-81100/

We should take a closer look at NR logs after deploy.

Reply:
We should look into migrating our custom integration with struclog then

Reply:
Yes. I read the release notes and also the documentation but I didn't find a HowTo guide or similar. I was thinking that maybe it's "automatic" 🤔  -- If you find any documentation, please let me know or open an issue.